tests/test_issues/test_linkml_issue_660.py

```diff
@@ -0,0 +1,89 @@
+import json
+import unittest
+
+from linkml.generators.jsonschemagen import JsonSchemaGenerator
+
+from tests.utils.test_environment import TestEnvironmentTestCase
+from tests.test_issues.environment import env
+
+# reported in https://github.com/linkml/linkml/issues/726
+
+schema_str = """
+id: http://example.org
+name: issue-726
+imports:
+  - https://w3id.org/linkml/types
+prefixes:
+  x: http://example.org/
+default_prefix: x
+default_range: string
+description: test
+classes:
+  C:
+    tree_root: true
+    slots:
+      - s1
+      - s2
+      - s3
+      - s4
+      - s5
+      - s6
+    slot_usage:
+      s1:
+        equals_string: foo
+      s3:
+        equals_number: 32
+      s5: 
+        if_absent: foo
+  D:
+    slots:
+      - s1
+      - s2
+      - s3
+      - s4
+slots:
+  s1:
+    description: test slot that can be overridden with specific values
+  s2:
+    equals_string: bar
+  s3:
+    description: test override for equals_number
+    range: integer
+  s4:
+    equals_number: 7
+    range: integer
+  s5: 
+    description: test override for if_absent
+    range: string
+  s6:
+    if_absent: foo
```

Comment:
https://w3id.org/linkml/ifabsent

      function that provides a default value for the slot.  Possible values for this slot are defined in
      linkml_runtime.utils.ifabsent_functions.default_library:
        * [Tt]rue -- boolean True
        * [Ff]alse -- boolean False
        * int(value) -- integer value
        * str(value) -- string value
        * default_range -- schema default range
        * bnode -- blank node identifier
        * slot_uri -- URI for the slot
        * class_curie -- CURIE for the containing class
        * class_uri -- URI for the containing class